client/app/bundles/shared/components/Radiobutton.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import css from './Radiobutton.scss';
+
+type Props = {
+  id?: string,
+  value?: string | number,
+};
+
+type State = {
+  value: string | number,
+  active: boolean
+};
+
+export default class Radiobutton extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { value: this.props.value || '', active: false };
+  }
+
+  onChange = (e: SyntheticEvent<HTMLInputElement>) => {
+    this.setState({ value: e.currentTarget.value });
+  };
+
+  render() {
+    const { id } = this.props;
+    return (
+      <div>
+        <form>
+          <label htmlFor={id} className={css.switch}> Option1
+            <input type="radio" value="option1" checked={this.state.value === 'option1'} onChange={this.onChange} />
+            <div className={css.check}>{}</div>
+          </label>
+          <label htmlFor={id} className={css.switch}>Option2
```

Comment:
Let's get rid of this second radio button, since the component should just be one radio button. If we want multiple on a page then we will do `<RadioButton etc. />` multiple times.